src/lib/InputProcessor.ts

```diff
@@ -80,3 +112,38 @@ function outputFormatFromInputs(inputs: Inputs): OutputFormat {
 		return OutputFormat.TABLE;
 	}
 }
+
+function trimMethodSpecifier(targetPath: string): string {
+	return targetPath.split('#')[0];
+}
+
+function getFirstCommonParentFolder(targetFiles: string[]) {
+	const longestCommonStr: string = getLongestCommonPrefix(targetFiles);
+	const commonParentFolder = getParentFolderOf(longestCommonStr);
+	return commonParentFolder.length == 0 ? '/' : commonParentFolder;
+}
+
+function getLongestCommonPrefix(strs: string[]): string {
```

Comment:
Could you add some commenting for the internals of this method? It's just complex enough that it's not immediately obvious how it works.

Reply:
Although you'll find that I am not a fan of unnecessary comments because they are yet another thing we need to maintain, if it helps in this case I will.

Reply:
Fixed.